packages/db/src/search/field-mapping.ts

```diff
@@ -249,6 +249,9 @@ const providerMapping : { [key: string]: Function } = {
 };
 
 const ruleMapping : { [key: string]: Function } = {
+  arn: (value?: string) => ({
```

Comment:
I think these single-word mappings might be unnecessary 🤔. If the property can be directly used as a postgres column, I probably don't need these...

Reply:
Turns out they are actually required, even if we're mapping e.g. `arn` to `arn` as-is. That's maybe something that could be refactored but leaving it to match the other record types here.